jaeger-zipkin/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     compile group: 'io.zipkin.reporter2', name: 'zipkin-reporter', version: '2.6.0'
 
     compile project(path: ':jaeger-core', configuration: 'shadow')
```

Comment:
Is a shadow configuration still required here? It might need to be on the jaeger-thrift dependency?